src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,5 @@
+##changeLogFile: changelog.sql
```

Comment:
Remove this file from the build.  It gets included in the built extension jar file.